content/en/blog/_posts/2019-06-13-volume-cloning-alpha.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: blog
+title: 'Introducing Volume Cloning Alpha for Kubernetes'
+date: 2019-06-13
+---
+
+**Author**: John Griffith (Red Hat)
+
+Kubernetes v1.15 introduces alpha support for volume cloning. This feature allows you to create new volumes using the contents of existing volumes in the users namespace using the Kubernetes API.
+
+## What is a Clone?
+
+Many storage systems provide the ability to to create a "clone" of a volume.  A clone is a duplicate of an existing volume that is its own unique volume on the system, but is an exact duplicate of in terms of its contents.  A clone is similar to a snapshot in that it's a point in time copy of a volume, however rather than creating a new snapshot object from a volume, we're instead creating a new independent volume, sometimes thought of as pre-populating the newly created volume.
+
+## Why add cloning to Kubernetes
+
+The Kubernetes volume plugin system already provides a powerful abstraction that automates the provisioning, attaching, and mounting of block and file storage.
+
+Underpinning all these features is the Kubernetes goal of workload portability: Kubernetes aims to create an abstraction layer between distributed systems applications and underlying clusters so that applications can be agnostic to the specifics of the cluster they run on and application deployment requires no “cluster specific” knowledge.
+
+The [Kubernetes Storage SIG](https://github.com/kubernetes/community/tree/master/sig-storage) identified clone operations as critical functionality for many stateful workloads. For example, a database administrator may want to duplicate a database volume and create another instance of an existing database.
+
+By providing a standard way to trigger clone operations in the Kubernetes API, Kubernetes users can now handle use cases like this without having to go around the Kubernetes API (and manually executing storage system specific operations).  While cloning is similair in behavior to creating a snapshot of a volume, then creating a volume from the snapshot, it's significantly more streamlined and in cases of most storage devices it is significantly more efficient for the device itself.  It also eliminates the need for another object to manage and performs the process in a single operation. 
```

Comment:
typo: "similair"

Reply:
maybe simplify to "the cloning operation is more streamlined and efficient for most storage systems"